microsoft/testsuites/performance/storageperf.py

```diff
@@ -190,6 +190,11 @@ def perf_storage_over_nfs_synthetic_udp_4k(self, result: TestResult) -> None:
         """,
         priority=3,
         timeout=TIME_OUT,
+        requirement=node_requirement(
+            node=schema.NodeSpace(
+                memory_mb=search_space.IntRange(min=4 * 1024),
```

Comment:
How did you arrive at this number?

Reply:
There is no system resource suggestion on [FIO Man page ](https://linux.die.net/man/1/fio) and FIO [readdocs](https://fio.readthedocs.io/en/latest/fio_doc.html). We have faces OOM with 512 MB memory. We deploy host azure VM with 64 GB so i kept 4GB requirement where FIO is running.

Reply:
> There is no system resource suggestion on [FIO Man page ](https://linux.die.net/man/1/fio) and FIO [readdocs](https://fio.readthedocs.io/en/latest/fio_doc.html). We have faces OOM with 512 MB memory. We deploy host azure VM with 64 GB so i kept 4GB requirement where FIO is running.

Good explanation, please add it to test case description.

Reply:
@anirudhrb, I will make RAM 2 GB as we discussed

Reply:
> There is no system resource suggestion on [FIO Man page ](https://linux.die.net/man/1/fio) and FIO [readdocs](https://fio.readthedocs.io/en/latest/fio_doc.html). We have faces OOM with 512 MB memory. We deploy host azure VM with 64 GB so i kept 4GB requirement where FIO is running.

Sure Chi

Reply:
@anirudhrb  @squirrelsc I have made changes as per our discussion

@LiliDeng I made changes for commit history

@anirudhrb Thnx for the help